addons/captives/stringtable.xml

```diff
@@ -136,6 +136,23 @@
             <Chinesesimp>将俘虏带出载具</Chinesesimp>
             <Turkish>Tutukluyu indir</Turkish>
         </Key>
+        <Key ID="STR_ACE_Captives_BlindfoldCaptive">
+            <English>Blindfold Captive</English>
+            <German>Augen verbinden</German>
+            <Spanish></Spanish>
```

Comment:
If you don't have a translation just don't make the key, it'll default to English. This will show an empty string.

Reply:
I planned on google translating those other than the 3 languages I speak.